packages/airbnb-base/test/airbnb_test.js

```diff
@@ -56,3 +56,169 @@ test('exposes eslintrc method', t => {
+        ecmaVersion: 2018,
+        sourceType: 'module'
+      },
+      plugins: ['babel'],
+      root: true,
+      rules: {
+        'babel/new-cap': [
+          'error',
+          {
+            capIsNew: false,
+            capIsNewExceptions: [
+              'Immutable.Map',
+              'Immutable.Set',
+              'Immutable.List'
+            ],
+            newIsCap: true,
+            newIsCapExceptions: []
+          }
+        ],
+        'babel/no-invalid-this': 'off',
+        'babel/no-unused-expressions': [
+          'error',
+          {
+            allowShortCircuit: false,
+            allowTaggedTemplates: false,
+            allowTernary: false
+          }
+        ],
+        'babel/object-curly-spacing': ['error', 'always'],
+        'babel/semi': ['error', 'always'],
+        'new-cap': 'off',
+        'no-invalid-this': 'off',
+        'no-unused-expressions': 'off',
+        'object-curly-spacing': 'off',
+        semi: 'off'
+      },
+      settings: {}
+    },
+    cache: true,
+    cwd: api.options.root,
+    failOnError: true,
+    formatter: require.resolve('eslint/lib/formatters/codeframe'),
+    useEslintrc: false
+  });
+});
+
+test('merges options with defaults', t => {
+  const api = new Neutrino();
+  api.use(mw(), {
+    test: /\.js$/,
+    include: ['/app/src'],
+    exclude: [/node_modules/],
+    eslint: {
+      baseConfig: {
+        extends: ['eslint-config-splendid'],
+        globals: {
+          jQuery: true
+        },
+        plugins: ['jest'],
+        rules: {
+          'babel/no-unused-expressions': 'warn'
+        },
+        settings: {
+          react: {
+            version: '16.5'
+          }
+        }
+      },
+      reportUnusedDisableDirectives: true
+    }
+  });
+
+  const lintRule = api.config.module.rule('lint');
+  t.deepEqual(lintRule.get('test'), /\.js$/);
+  t.deepEqual(lintRule.include.values(), ['/app/src']);
+  t.deepEqual(lintRule.exclude.values(), [/node_modules/]);
+  t.deepEqual(lintRule.use('eslint').get('options'), {
+    baseConfig: {
+      env: {
+        es6: true
+      },
+      extends: [
+        require.resolve('eslint-config-airbnb-base'),
+        'eslint-config-splendid'
+      ],
+      globals: {
+        jQuery: true,
+        process: true
+      },
+      overrides: [],
+      parser: require.resolve('babel-eslint'),
+      parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: 'module'
+      },
+      plugins: [
+        'babel',
+        'jest'
+      ],
+      root: true,
+      rules: {
+        'babel/new-cap': [
+          'error',
+          {
+            capIsNew: false,
+            capIsNewExceptions: [
+              'Immutable.Map',
+              'Immutable.Set',
+              'Immutable.List'
+            ],
+            newIsCap: true,
+            newIsCapExceptions: []
+          }
+        ],
+        'babel/no-invalid-this': 'off',
+        'babel/no-unused-expressions': [
+          'warn',
```

Comment:
This case is an example of one of the tests that would have failed previously - since before the resultant config would have discarded the rule settings, and just had the value of `'warn'`.